test/functional/core/job_spec.lua

```diff
@@ -124,17 +124,26 @@ describe('jobs', function()
   end)
 
   it('allows interactive commands', function()
-    if helpers.pending_win32(pending) then return end  -- TODO: Need `cat`.
-    nvim('command', "let j = jobstart(['cat', '-'], g:job_opts)")
-    neq(0, eval('j'))
-    nvim('command', 'call jobsend(j, "abc\\n")')
-    eq({'notification', 'stdout', {0, {'abc', ''}}}, next_msg())
-    nvim('command', 'call jobsend(j, "123\\nxyz\\n")')
-    eq({'notification', 'stdout', {0, {'123', 'xyz', ''}}}, next_msg())
-    nvim('command', 'call jobsend(j, [123, "xyz", ""])')
-    eq({'notification', 'stdout', {0, {'123', 'xyz', ''}}}, next_msg())
-    nvim('command', "call jobstop(j)")
-    eq({'notification', 'exit', {0, 0}}, next_msg())
+    if iswin() then
+      helpers.reset_shell_opts()
+      nvim('command', 'set shellcmdflag^=/v:ON\\ ')
+      nvim('command', "let j = jobstart('set /p abc= & echo !abc!', g:job_opts)")
```

Comment:
This hack works only in cmd.exe. Unless someone has a better idea on how to do this in powershell with `jobsend`, I will use this for the rest of pending job tests in Windows.

Reply:
`findstr "^"` is an alternative for `cat -` but does not require delayed expansion. Like `cat`, it requires `jobstop` or Ctrl-C in the cmd.exe terminal to kill it.

Reply:
@janlazo  POSIX-like `cat` (`cat.exe`) is available on Windows (in our test system/appveyor, and also shipped to users). Does that help anything?

Reply:
Yes. That simplifies the test cases. It helps that `jobstart(['cat', '-'], g:job_opts)` gets around the `cat` alias from powershell.

Reply:
ok, `cat -` is used in `system_spec.lua` in tests that pass on appveyor.

Note sure why that old `TODO: Need `cat`.` note was still in `job_spec.lua`.